### PR TITLE
feat: update python3-ifcfg key

### PR DESCRIPTION
**Pull request by wep21**
Signed-off-by: wep21 <border_goldenmarket@yahoo.co.jp>

Please add the following dependency to the rosdep database.

## Package name:

python3-ifcfg

## Package Upstream Source:

https://github.com/osrf/python-ifcfg-release

## Purpose of using this:

add latest distribution

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - REQUIRED
- Ubuntu: https://packages.ubuntu.com/
   - REQUIRED
- Fedora: https://src.fedoraproject.org/browse/projects/
  - IF AVAILABLE
- Arch: https://www.archlinux.org/packages/
  - IF AVAILABLE
- Gentoo: https://packages.gentoo.org/
  - IF AVAILABLE
- macOS: https://formulae.brew.sh/
  - IF AVAILABLE
- Alpine: https://pkgs.alpinelinux.org/packages
  - IF AVAILABLE
- NixOS/nixpkgs: https://search.nixos.org/packages
  - OPTIONAL

# Please Add This Package to be indexed in the rosdistro.

ROSDISTRO NAME

# The source is here: 

http://sourcecode_url

# Checks
 - [ ] All packages have a declared license in the package.xml
 - [ ] This repository has a LICENSE file
 - [ ] This package is expected to build on the submitted rosdistro

### CODE REVIEW

**Review by hidmic — COMMENTED** [review]
@wep21 hmm, I don't see these packages in [`packages.ros.org`](http://packages.ros.org/ros2/ubuntu/dists/jammy/main/binary-amd64/Packages). 

@nuclearsandwich is there an timeline for this?

File: rosdep/python.yaml [hunk 1]
```diff
@@ -6634,6 +6634,7 @@ python3-hypothesis:
   ubuntu: [python3-hypothesis]
 python3-ifcfg:
   debian:
+    bullseye: [python3-ifcfg]
     buster: [python3-ifcfg]
     stretch: [python3-ifcfg]
```
Comment (clalancette) [inline rosdep/python.yaml:6639]: I think that we should just change this to a generic key for all debian versions:

```
debian: [python3-ifcfg]
```

That will make it future-proof.  The same goes for the Ubuntu rule below.

**Review by clalancette — CHANGES_REQUESTED** [review]

**Comment by nuclearsandwich** [comment]
> @nuclearsandwich is there an timeline for this?

We've been trying to remove this package from the ROS bootstrap repository and push it upstream into Debian and Ubuntu.
During review, the package was dismissed and after some discussion @clalancette has been working to enhance the network configuration APIs in [psutil](https://pypi.org/project/psutil/) to provide what we need for ros2doctor and eliminate the ifcfg dependency.

In short, providing python3-ifcfg for Ubuntu Jammy and Debian Bullseye is unplanned. We're removing that dependency from ros2doctor in favor of psutil.

File: rosdep/python.yaml (hunk 1 above)
Comment (wep21) [reply rosdep/python.yaml]: addressed at https://github.com/ros/rosdistro/pull/31482/commits/cdc26081043a7bd054869ab7b4caeca2daf0052d.

**Comment by wep21** [comment]
@nuclearsandwich I see. I close this PR.

**Pull request closed without merging**